color/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="md_dialog_color_presets">Presets</string>
```

Comment:
We'd probably want to allow the extension method to accept custom strings for these values, e.g. for translations from consuming apps